src/main/java/blusunrize/immersiveengineering/common/IERecipes.java

```diff
@@ -521,7 +565,7 @@ public static ArcFurnaceRecipe addArcRecipe(ItemStack output, Object input, int
 
 	public static void addArcOreSmelting(ItemStack output, String ore)
 	{
-		if(ApiUtils.isNonemptyItemTag(getOre(ore)))
+		if(ApiUtils.isNonemptyItemTag(getOre(ore)))//ToDo this should allow blocks too
```

Comment:
Why? Every block tag should have an equivalent item tag.

Reply:
See my fix in src/main/java/blusunrize/immersiveengineering/api/ApiUtils.java#82

It doesn't seem to always be cohesive?